requirements.txt

```diff
@@ -15,6 +15,7 @@ statsmodels
 sympy
 tensorflow
 texttable
+torch
```

Comment:
https://github.com/pytorch/pytorch/issues/86566

Reply:
Do I need to do anything about this?

Reply:
The only thing we can do is wait until torch is compatible with the current version of Python.

Reply:
Okay, after it is compatible this PR will be automatically merged?